docs/articles/recommender_from_clickstreams.md

```diff
@@ -0,0 +1,214 @@
+# How to Build a Recommender System From Clickstream Data
+
+<!-- test comment -->
+
+Whenever you have a website with products that customers can browse, you can hardly save yourself from the flood of **clickstream data**. For example, a user of a hotel website might produce the following data when browsing different hotels:
+
+```mermaid
+graph LR;
+      id1[Tranquil Haven Resort]-->id2[Summit Vista Inn];
+      id2[Summit Vista Inn]-->id3[Azure Sands Hotel];
+      id3[Azure Sands Hotel]-->id4[Evergreen Retreat Lodge];
+```
+
+They start with the Tranquil Haven Resort, and end their browsing session with the Evergreen Retreat Lodge. We can do different things with this data now, one of them being finding **embeddings** for all hotels.
```

Comment:
but not "finding", rather "training" embeddings using the clickstream data, no?

Reply:
Hi! Both versions are kind of the same for me, but I can change it to creating.

Reply:
Ah, I see that you edited some things already, so you can change it to "creating" if you want, I don't have a strong opinion on that one.

Reply:
@Garve 
Yes, sorry, at the style edit stage, it typically takes a very long time to go back and forth with authors, indicating what should be changed, and waiting to see if they make those changes.
That being said, I keep my edits as minimal as possible, and will have you do a final pass to make sure everything's okay, before we publish.
In this case, "getting" would imply that the hotel embeddings are already there, waiting to be fetched.

Reply:
Fine for me!